traitlets/utils/decorators.py

```diff
@@ -0,0 +1,40 @@
+from .. import Undefined
+
+
+def enable_auto_completion(cls):
+    """Returns a decorator enabling auto-completion for the class traits"""
+
+    def format_value(value):
+        if value is Undefined:
+            return 'None'
```

Comment:
shouldn't we return 'traitlets.Undefined' instead?

Reply:
What is `traitlets` is not defined in the scope? It will raise an error